docs/index.rst

```diff
@@ -0,0 +1,33 @@
+################################
+Qiskit Algorithms documentation
+################################
+
+Overview
+========
+
+Qiskit Algorithms is a library of quantum algorithms based on Qiskit,
+suitable to run on near-term quantum devices with short-depth circuits.
+
+
+Next Steps
+===========
+
+`Getting started <getting_started.html>`_
+
+`Tutorials <tutorials/index.html>`_
+
+
+.. currentmodule:: qiskit_algorithms
+
+.. toctree::
+  :hidden:
+
+  API References <apidocs/algorithms>
```

Comment:
```suggestion
  About Qiskit Algorithms <self>
  Getting Started <getting_started>
  API References <apidocs/algorithms>
```

Reply:
I don't have a strong preference :) Others use `Documentation Home` for that reason. `Overview` works for me too.